views/userDetails.njk

```diff
@@ -276,7 +276,7 @@
           }) }}
         {% endif %}
 
-        {% if showCaseloads %}
+        {% if caseloads %}
```

Comment:
We now show caseloads if any are available, it is no longer based on whether they have an active caseload or not.  This could mean they allocate caseloads but none are their active caseload.  @Mjwillis is this ok?

Reply:
yes